### PR TITLE
Add search for sketchfab collections and tweak featured search params.

### PR DESCRIPTION
**Pull request by robertlong**

### CODE REVIEW

File: lib/ret/media_search.ex [hunk 1]
```diff
@@ -62,38 +63,89 @@ defmodule Ret.MediaSearch do
     favorites_search(cursor, type, account_id, q)
   end
 
+  def search(%Ret.MediaSearchQuery{source: "sketchfab", cursor: cursor, filter: nil, collection: nil, q: q}) do
+    if q == nil || q == "" do
```
Comment (gfodor) [inline lib/ret/media_search.ex]: this should be a separate match

File: lib/ret/media_search.ex [hunk 2]
```diff
@@ -62,38 +63,89 @@ defmodule Ret.MediaSearch do
...
     query =
       URI.encode_query(
         type: :models,
         downloadable: true,
         count: @page_size,
         max_face_count: @max_face_count,
+        max_filesizes: "gltf:#{@max_file_size_bytes}",
         processing_status: :succeeded,
+        sort_by:
+          if q == nil || q == "" do
```
Comment (gfodor) [inline lib/ret/media_search.ex]: if its a collection, should we care about the query here?

Comment (robertlong) [reply lib/ret/media_search.ex]: Yeah, you can search within a collection, which can be useful for larger collections.

File: lib/ret/media_search.ex (hunk 1 above)
Comment (robertlong) [reply lib/ret/media_search.ex]: How can I match on nil and ""?

Comment (gfodor) [reply lib/ret/media_search.ex]: you can use `when` https://hexdocs.pm/elixir/master/guards.html

File: lib/ret/media_search.ex (hunk 2 above)
Comment (gfodor) [reply lib/ret/media_search.ex]: sorry, what i meant is if the `sort_by` should be `nil` if we have a query. i guess that seems right assuming sketchfab will sort by relevance.

Comment (robertlong) [reply lib/ret/media_search.ex]: Yeah, I'm pretty sure it does.

**Pull request merged**